src/main/webapp/populateData.js

```diff
@@ -102,17 +102,18 @@ function populateCalendar() {
         }
         return response.json();
       })
-      .then((eventList) => {
-        // Convert JSON to string containing all event summaries
-        // and display it on client
-        // Handle case where user has no events to avoid unwanted behaviour
-        if (eventList.length !== 0) {
-          const events =
-              eventList.map((a) => a.summary).reduce((a, b) => a + '\n' + b);
-          calendarContainer.innerText = events;
-        } else {
-          calendarContainer.innerText = 'No events in the calendar';
+      .then((hoursJson) => {
```

Comment:
`mvn spotless:apply` doesn't deal with linting JavaScript so make sure you manually lint according to the suggestions